.env.example

```diff
@@ -159,3 +159,9 @@ TIMEZONE="Pacific Time (US & Canada)"
 
 # Number of failed jobs to keep in the database
 FAILED_JOBS_TO_KEEP=100
+
+# Maximum runtime of background jobs in minutes
+# DELAYED_JOB_MAX_RUNTIME=20
+
+# Amount of seconds for delayed_job to sleep before checking for new jobs
+# DELAYED_JOB_SLEEP_DELAY=10
```

Comment:
I guess I've generally thought of enabling everything with the default, so
long as the code also has that default so that we can support old .env
files.  Any downside?

On Wednesday, May 13, 2015, Dominik Sander <notifications@github.com> wrote:

> In .env.example
> <https://github.com/cantino/huginn/pull/821#discussion_r30206492>:
>
> > @@ -159,3 +159,9 @@ TIMEZONE="Pacific Time (US & Canada)"
> >
> >  # Number of failed jobs to keep in the database
> >  FAILED_JOBS_TO_KEEP=100
> > +
> > +# Maximum runtime of background jobs in minutes
> > +# DELAYED_JOB_MAX_RUNTIME=20
> > +
> > +# Amount of seconds for delayed_job to sleep before checking for new jobs
> > +# DELAYED_JOB_SLEEP_DELAY=10
>
> Not really, do you think we should have every option 'enabled' with the
> default value? SCHEDULER_FREQUENCY is also commented out, but those three
> are the only ones
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cantino/huginn/pull/821/files#r30206492>.
>